src/xpcc/architecture/platform/driver/core/cortex/driver.xml

```diff
@@ -20,6 +20,10 @@
 		<parameter name="free_rtos_support" type="bool">false</parameter>
 		<parameter name="free_rtos_frequency" type="int" min="1000" max="10000">1000</parameter>
 
+		<parameter name="linkerscript_memory" type="string"></parameter>
+		<parameter name="linkerscript_sections" type="string"></parameter>
+		<parameter name="linkerscript_table_zero_extern" type="string"></parameter>
+		<parameter name="linkerscript_table_copy_extern" type="string"></parameter>
```

Comment:
Exactly, but I wasn't going to write a grammar for linker script language 😛.
Should I call this `rawstring` or `raw` instead?